components/Tour/StageCard.tsx

```diff
@@ -50,6 +51,7 @@ export default function StageCard({ stage }: { stage: Stage }) {
           ...styles.card,
         }}
         header={<Header />}
+        status={stage.isActive ? "primary" : ""}
```

Comment:
```suggestion
        status={stage.isActive ? "primary" : undefined}
```
Würde lieber auf undefined setzten, als auf einen leeren String, weil die Option Optional ist.